web/client/components/TOC/fragments/settings/Display.jsx

```diff
@@ -261,7 +264,22 @@ export default class extends React.Component {
                         <Col xs={12} className={"legend-label"}>
                             <label key="legend-options-title" className="control-label"><Message msgId="layerProperties.legendOptions.title" /></label>
                         </Col>
-                        <Col xs={12} sm={6} className="first-selectize">
+                        { this.props.element?.serverType !== ServerTypes.NO_VENDOR &&
+                            <Col xs={12}>
```

Comment:
the checkbox is not aligned to the other

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/ff329377-d91c-4a6b-96a3-8d2a4d290e11)

Reply:
@allyoucanmap @mahmoudadel54 let's put a proper info message in the helper to well inform what the option does and what it requires. For layers not coming from GeoServer this option should be disabled or simply not available.